front/src/common/redux/root-reducer.js

```diff
@@ -1,10 +1,10 @@
 import { combineReducers } from 'redux';
 import { studentsReducer } from './students/students.reducer';
 import { login, changePassword } from './auth/auth.reducer';
-import { groupList } from './groups/groups.reducer';
+import { groupList, createGroupReducer } from './groups/groups.reducer';
```

Comment:
Please, do a little favor, remove `Reducer` suffix from all the names of reducers

Reply:
done